### DISCUSSION TITLE
en/latest/UserGuide/%E5%A6%82%E4%BD%95%E4%B8%8B%E8%BD%BD/

### DISCUSSION THREAD

**Discussion by giscus** [zdsjjtTLG/TrackIt#18] [General]
# en/latest/UserGuide/%E5%A6%82%E4%BD%95%E4%B8%8B%E8%BD%BD/

在python中轻松进行路径匹配

https://gotrackit.readthedocs.io/en/latest/UserGuide/%E5%A6%82%E4%BD%95%E4%B8%8B%E8%BD%BD/

**Comment by wuhao-ai** [comment]
error: subprocess-exited-with-error

  × pip subprocess to install build dependencies did not run successfully.
  │ exit code: 1
  ╰─> [102 lines of output]
安装keplergl出现这个错误是为什么？

**Reply by zdsjjtTLG** [reply]
网络问题吧
